components/file_preview/file_preview.tsx

```diff
@@ -110,16 +121,21 @@ export default class FilePreview extends React.PureComponent {
             );
         });
 
-        this.props.uploadsInProgress.forEach((clientId) => {
-            previews.push(
-                <FileProgressPreview
-                    key={clientId}
-                    clientId={clientId}
-                    fileInfo={this.props.uploadsProgressPercent[clientId]}
-                    handleRemove={this.handleRemove}
-                />
-            );
-        });
+        if (this.props.uploadsInProgress) {
+            this.props.uploadsInProgress.forEach((clientId) => {
+                if (this.props.uploadsProgressPercent) {
```

Comment:
Since this doesn't depend on an iteration of the loop, you can null-check this outside of the loop.